.cirrus.yml

```
@@ -0,0 +1,565 @@
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "bazarr.json"
+
+bind_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "bind.json"
+
+calibre-web_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-2
+  env:
+    PLUGIN_FILE: "calibre-web.json"
+
+channels-dvr_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "channels-dvr.json"
+
+clamav_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "clamav.json"
+
+deluge-pip_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-2
+  env:
+    PLUGIN_FILE: "deluge-pip.json"
+
+dnsmasq_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "dnsmasq.json"
+
+drupal8_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "drupal8.json"
+
+duplicati_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "duplicati.json"
+
+emby_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "emby.json"
+
+emby-server-stable_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-2
+  env:
+    PLUGIN_FILE: "emby-server-stable.json"
+
+esphome_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-3
+  env:
+    PLUGIN_FILE: "esphome.json"
+
+famp_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "famp.json"
+
+gitea_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "gitea.json"
+
+gitlab_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "gitlab.json"
+
+gogs_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "gogs.json"
+
+grafana_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "grafana.json"
+
+heimdall-dashboard_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
+  env:
+    PLUGIN_FILE: "heimdall-dashboard.json"
+
+homeassistant_task:
+  <<: *INSTALL_PLUGIN
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-11-2
+    - freebsd_instance:
+        image_family: freebsd-11-3
+    - freebsd_instance:
+        image_family: freebsd-12-1
```

I'm surprised this even passed the test on an 11.x

However, as the plugin's maintainer, I am not long supporting the Home Assistant plugin on an 11.x release

```suggestion
    - freebsd_instance:
        image_family: freebsd-12.1
    - freebsd_instance:
        image_family: freebsd-12-2
```

---

Added this one as a debug step, because I got a bit confused why it was failing myself. Should be removed and match the plugin JSON file now.